src/WrappedMToken.sol

```diff
@@ -63,6 +64,9 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
     /// @inheritdoc IWrappedMToken
     address public immutable vault;
 
+    /// @inheritdoc IWrappedMToken
+    address public immutable claimRecipientManager;
```

Comment:
Don't we need to add `claimRecipientManager` after `uint128[] internal _enableDisableEarningIndices;` to not affect the storage layout?

Reply:
It's immutable so it doesn't occupy any storage slots.